.cursor/commands/spec-document-creator.md

```
@@ -241,67 +256,83 @@ After extraction:
 ```
 
 ## Authentication
+
 Authentication requirements.
 
 ## Error Handling
+
 Error response format.
-```
+
+````
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)